packages/block-editor/src/components/inspector-controls/slot.js

```diff
@@ -34,6 +34,7 @@ export default function InspectorControlsSlot( {
 			<BlockSupportToolsPanel group={ group } label={ label }>
 				<BlockSupportSlotContainer
 					{ ...props }
+					className="components-tools-panel__items-wrapper"
```

Comment:
Why is this className necessary?

Reply:
Good catch, thank you!

I don't believe it is needed anymore, I missed this while repeatedly rebasing this chain of PRs. Thankfully it's not 3 or 4 deep now. 😅 

The class was removed in https://github.com/WordPress/gutenberg/pull/34725/commits/e5809a887ef94fa9b90f896d6f3436fa3e325da7 as part of the work making the grouped `InspectorControls` slots bubble virtually for block supports.